sherpa/image/DS9.py

```diff
@@ -237,8 +237,30 @@ def __init__(self, *args, **kargs):
 
         if doRaise:
             raise RuntimeErr('badwin', _ex)
+
+    elif sys.version_info < (3, 2):
```

Comment:
I don't mind using `sys.version_info`, it will also be easier to spot these changes if and when we decide not to support Python 2 any more.